modules/mintmaker/pages/default-config.adoc

```diff
@@ -0,0 +1,92 @@
+= Default configuration
+
+MintMaker applies a set of default options for all repositories on backend.
+The configuration file can be found https://github.com/konflux-ci/mintmaker/blob/main/config/renovate/renovate.json[here].
+
+In this page you will find a summary of the config options and decisions behind them.
+
+For details on individual manager implementation, version support or limitations,
+proceed to xref:mintmaker:support.adoc[].
+
+== Renovate presets
+
+- https://docs.renovatebot.com/presets-config/#configrecommended[`config:recommended`] as the base template recommended by Renovate for most users
+- https://docs.renovatebot.com/presets-default/#gitsignoff[`:gitSignOff`] - adds the `Signed-off-by: <author>` part to commit messages
+- https://docs.renovatebot.com/presets-default/#disabledependencydashboard[`:disableDependencyDashboard`] - can be enabled individually in user config
+
+== Configuration
+
+MintMaker is set up to not require any additional onboarding (only in Konflux)
+or configuration files - it works even without the `renovate.json` file present.
+
+== Scheduling
+
+Please refer to the main guide on scheduling xref:mintmaker:user.adoc#scheduling[here].
+
+By default, MintMaker will not update PRs outside of the default schedule,
+due to `updateNotScheduled` being `false`. More on this behavior file:///home/riley/projects/personal/docs/public/mintmaker/user/index.html#how-to-stop-prsmrs-from-being-updated-outside-of-schedule[here].
+
+== Branch limits
+
+We set `branchConcurrentLimit` to 0 and `pruneStaleBranches` to `false` due
+to the way how MintMaker runs Renovate jobs against Konflux components.
+A single repository/branch can be accessed by multiple jobs and without
+this configuration, Renovate would close and reopen PRs every time it runs.
+
+== Vulnerability alerts
+
+We enable vulnerability alerts, but include two custom options:
+
+- `containerVulnerabilityAlerts`
+- `rpmVulnerabilityAlerts`
+
+This serves to differentiate container and RPM xref:mintmaker:security-updates.adoc[security updates] from the other ecosystems.
+
+== Branch naming convention
+
+MintMaker sets a prefix `konflux/mintmaker/{{ baseBranch }}` for all its branches.
+
+== Enabled managers
+
+MintMaker enables a list of managers by default. The full list is accessible
+xref:mintmaker:user.adoc#available-managers[here].
+
+== Tekton updates
+
+The `tekton` manager is pointed to the `.tekton/` folder and enabled
+for `quay.io/redhat-appstudio-tekton-catalog` and `quay.io/konflux-ci/tekton-catalog` images. It also includes links for migration guides
```

Comment:
iiuc, it supports all images used in Tekton pipelines. However, only images from `quay.io/redhat-appstudio-tekton-catalog` and `quay.io/konflux-ci/tekton-catalog` repositories are grouped.

This reminds me one case we might not considered : what will happen when users add custom tasks with their own container images, and post upgrade task is executed against these custom tasks? @tkdchen

Reply:
Thanks, reworded, does it sound better now?